dev-docs/modules/contxtfulRtdProvider.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page_v2
+title: Contxtful RTD Provider  
+display_name: Contxtful RTD Provider  
+description: Activates Receptivity to evaluate ad success in real-time by focusing on attention and context.
+page_type: module
+module_type: rtd
+module_code : contxtfulRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# Contxtful Real Time Data Module
+
+## Overview
+
+The Contxtful RTD module offers a unique feature—Receptivity. Receptivity is an efficiency metric, enabling the qualification of any instant in a session in real time based on attention. The core idea is straightforward: the likelihood of an ad’s success increases when it grabs attention and is presented in the right context at the right time.
```

Comment:
The prebid documentation is a technical documentation and thus requires no marketing language. Specify what your module technically does (adding meta information to a prebid ad unit ).

You should also add the disclaimer that this script loads external javascript

```liquid
{% include dev-docs/loads-external-javascript.md %}
```

Reply:
Thanks for this feedback!
I revised the documentation and included the disclaimer accordingly; please check it.

Reply:
@muuki88 Hi !
I tried to include the template but I get an error on the CI [here](https://app.netlify.com/sites/prebid-docs-preview/deploys/6519db8530cc6300088f0ed7)
> Could not locate the included file "dev-docs/loads-external-javascript.md"
Do you know what's the proper syntax to use it?
Thanks a lot.

Reply:
Hey @muuki88 Thanks again for the suggestion!
As I am blocked by this -- Do you know why it is failing at the CI level? Should I use other way?

As a workaround only, I could also copy / paste the content of that [markdown](https://github.com/prebid/prebid.github.io/blob/e97abc66f8600772191fe06d782db81803df5b7d/_includes/dev-docs/loads-external-javascript.md#L2) although it would not follow the DRY principle. 

Thanks!

Reply:
I'm looking into this when I'm home.

The PR will be merged once the JS module is released